pkg/kudoctl/cmd/plan/plan_status_test.go

```diff
@@ -82,7 +82,7 @@ func TestStatus(t *testing.T) {
 .
 └── test (Operator-Version: "test-1.0" Active-Plan: "deploy")
     └── Plan deploy ( strategy) [FATAL_ERROR]
-        └── Phase deploy [FATAL_ERROR]
+        └── Phase deploy ( strategy) [FATAL_ERROR]
```

Comment:
this is not a very nice output :/ but I guess it's fine for now, I'll fix it once we merge it :)

This actually should never happen in real life... ? because the strategy should is mandatory in OV